README.md

```diff
@@ -41,15 +41,14 @@ or in /opt/trinity/tc-portal-sync if installing via the RPM.
 
 The "trinity_cyber_portal" section of the configuration file contains application-level settings:
 
-| Field                 | Description                                                                                                                        |
-|-----------------------|------------------------------------------------------------------------------------------------------------------------------------|
-| api_url               | The URL for the Trinity Cyber Portal GraphQL endpoint. Keep the default unless accessing a custom portal instance.                 |
-| api_key               | API key. This can be generated from your user profile menu (top right) on https://portal.trinitycyber.com                          |
-| marker_file           | The file used to keep track of current synchronization state.                                                                      |
-| customer_gids         | (For MSP users) Specifies which customers' events to download. The values can be found by running "tc_portal_sync.py --customers"  |
-| poll_interval_seconds | How long to wait (in seconds) between API calls when no additional events are available.                                           |
-| query_name            | Update with a descriptive query name that can be used for support purposes. Example: MyCompanyMyApplication                        |
-
+| Field                 | Description                                                                                                      |
+|-----------------------|------------------------------------------------------------------------------------------------------------------|
+| api_url               | The URL for the Trinity Cyber Portal GraphQL endpoint. Keep the default unless accessing a custom portal instance. |
```

Comment:
would like to see the end of line formatting remain consistent.  Would almost say pick a width and that is now fixed, lines wrap if necessary.